### PR TITLE
[Misc] Update CREDITS.md

### PR DESCRIPTION
**Pull request by DayKev**
@muscode13 @geeilhan
Let me know if you'd rather be credited with a different or additional [user]name.

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -373,62 +373,64 @@ In addition to the lists below, please check [the PokéRogue wiki](https://wiki.
 - PigeonBar
 
 ## Past Contributors
```
Comment (Tempo-anon) [inline CREDITS.md:375]: Wonder if this should be renamed to something like `Other Code Contributors` as there are also contributors that are currently contributing. This is just a catchall section for code contributors that aren't officially part of the dev team

Comment (DayKev) [inline CREDITS.md:375]: Sure.
```suggestion
## Other Code Contributors
```

File: CREDITS.md [hunk 2]
```diff
@@ -372,63 +372,65 @@ In addition to the lists below, please check [the PokéRogue wiki](https://wiki.
...
-- francktrouillez
 - JakubHanko
-- FredeX
-- PigeonBar
-- prime-dialga
-- rnicar245
-- rationality6
+- JonStudders
+- karl-police
+- lucfd
```
Comment (MokaStitcher) [inline CREDITS.md:403]:
```suggestion
- lucfd
- Lugiadrien
```
also contributed outside of pure translation work (GH username Adri1, but putting this one for consistency with name used in rest of credits)

File: CREDITS.md [hunk 3]
```diff
@@ -372,63 +372,65 @@ In addition to the lists below, please check [the PokéRogue wiki](https://wiki.
...
+- schmidtc1
+- shayebeadling
+- sirzento
+- snoozbuster
+- sodaMelon
+- td76099
+- Vassiat
+- Xiaphear
+- zaccie
+- zacharied
```
Comment (MokaStitcher) [inline CREDITS.md:433]:
```suggestion
- zacharied
- Zé Ricardo
```
also contributed outside of pure translation work (candy notification in starter select, maybe other things too) (GH username josericardo-fo, putting this name for consistency with rest of the credits)

**Review by MokaStitcher — COMMENTED** [review]
double checked through every contributors in the past 3 months (on the GH page, so people may still be missing) and found a couple that are credited for their translation work but not dev work

**Review by MokaStitcher — APPROVED** [review]

**Pull request merged**